scripts/pytheta.py

```diff
@@ -106,14 +106,14 @@ def check_if_theta(xtp_dev_list):
 
 		print('[debug] [{}]'.format(res))
 		if re.match(".*RICOH THETA.*",res):
-			print("[debug]シータです")
+			print("[debug] Found THETA camera. [シータです]")
```

Comment:
So the print statement will show both English and Japanese?

Reply:
it does now

Reply:
OK, understood.